requirements/pip.txt

```diff
@@ -0,0 +1,4 @@
+# Dependancies for setting up pip to install our requirements.txt file.
+setuptools==35.0.2
+pip==9.0.1
+wheel==0.29.0
```

Comment:
Pinned versions for `pip` and `wheel` are there latest versions which have been used till now. I just pinned them so from now on we will need to modify them manually. This is to avoid painful situation updating to some buggy version.